db/migrate/20221122123446_add_user_to_group.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToGroup < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :groups, :user, null: false, foreign_key: {to_table: :users}
+  end
```

Comment:
- [ ] The Entity Relationship diagram shows that the name of a user should be an author. In this case, the foreign key should be `author_id` instead of `user_id`

Reply:
Hello @elisha2kyakpo1, thank you for the review.
Please note that the user_id and author_id is exactly the same thing which is the id of the current user , which is the person who has logged in.

Reply:
@aneallaryea100 it's true that user_id and author_id is the same, but we have to explicitly indicate that the user is an author or whatever the requirement is.